src/app/custom-navigation-mode/custom-navigation-mode.ts

```diff
@@ -2,6 +2,8 @@ import {WizardComponent, ConfigurableNavigationMode} from 'angular-archwizard';
 
 /**
  * A custom NavigationMode which allows to navigate backward to any step and forward to steps the user has already visited before.
+ *
+ * This class only exists for demonstration purposes.  In a real application, one should simple use `[navigateForward]="visited"`.

```

Comment:
I think you should also add a short note the html file informing the users that this wizard behavior can also be achieved using `[navigateForward]="visited"`

Reply:
Agree. Note added in 8edc809